browser/net/brave_ad_block_tp_network_delegate_helper_unittest.cc

```diff
@@ -25,10 +71,40 @@ TEST(BraveAdBlockTPNetworkDelegateHelperTest, NoChangeURL) {
   EXPECT_EQ(rc, net::OK);
 }
 
-TEST(BraveAdBlockTPNetworkDelegateHelperTest, EmptyRequestURL) {
+TEST_F(BraveAdBlockTPNetworkDelegateHelperTest, EmptyRequestURL) {
   auto request_info = std::make_shared<brave::BraveRequestInfo>(GURL());
   int rc =
       OnBeforeURLRequest_AdBlockTPPreWork(ResponseCallback(), request_info);
   EXPECT_TRUE(request_info->new_url_spec.empty());
   EXPECT_EQ(rc, net::OK);
 }
+
+TEST_F(BraveAdBlockTPNetworkDelegateHelperTest, ByassTor) {
```

Comment:
ByPassTor

Reply:
actually I think this is a bad name because it makes it sound like it should be bypassing Tor and that's exactly the opposite of what we are doing. Maybe something like DisableInsecureRequestsOverTor

Reply:
fixed